websites/D/Discord.place/metadata.json

```
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "682607343707488388",
+		"name": "nego"
+	},
+	"service": "Discord.place",
+	"description": {
+		"en": "A place for all things that related to Discord. No matter if you are a developer, a server owner, or just a user, you can find something useful here."
+	},
+	"url": "discord.place",
+	"version": "1.1.2",
```

Since this is a new presence, the version should be `1.0.0`
```suggestion
	"version": "1.0.0",
```